.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug Asterinas(debug)",
+            "type": "lldb", // install CodeLLDB extension first
+            "request": "custom",
+            "targetCreateCommands": ["target create ${workspaceFolder}/target/x86_64-custom/debug/asterinas"],
+            "processCreateCommands": ["gdb-remote 1234"],
```

Comment:
Where does this magic number of 1234 come from?

Reply:
`-s` equals to `-gdb tcp::1234`, i.e. open a gdbserver on TCP port 1234.